db/migrations/0007_add_owner_urls.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS owner_urls (
+  owner TEXT NOT NULL,
+  pdp_url TEXT NOT NULL,
```

Comment:
IIRC, there are two URLs in the Pandora registry of SPs: pdpUrl is used for uploading new data and pieceRetrievalUrl is used for retrievals.

See https://github.com/FilOzone/filecoin-services/blob/1b99d8b39bf7f35608884b0bd7bd768810ee2372/service_contracts/src/PandoraService.sol#L119

For FilCDN, we need the latter.

Let's name the column accordingly to prevent confusion.

```suggestion
  piece_retrieval_url TEXT NOT NULL,
```